surveyPage5.php

```diff
@@ -10,84 +10,148 @@
     <body>
         <br><br><br><br>
         <div class="question-container">
-            <h4><center>Question </center><h4>
-            <h5><center>?</center><h5>
+            <h4><center>Question 46</center><h4>
+            <h5><center>Do you like plants?</center><h5>
                 <div>
-                    <input type="radio" value="" id="Q1A1" name="Q1"/>
-                    <label for="Q1A1">Answer</label>
+                    <input type="radio" value="" id="Q46A1" name="Q46"/>
```

Comment:
Make sure to fill in the value field to match the label !

Reply:
I think I fixed it thanks!